test/blockwise.js

```diff
@@ -6,23 +6,56 @@
  * See the included LICENSE file for more details.
  */
 
-var coap = require('../')
+var coap        = require('../')
+  , parse       = require('coap-packet').parse
+  , generate    = require('coap-packet').generate
+  , getOption   = require('../lib/helpers').getOption
+  , parseBlock2 = require('../lib/helpers').parseBlock2
+  , dgram       = require('dgram')
 
 describe('blockwise2', function() {
   var server
     , port
+    , clientPort
+    , client
+    , bufferVal
     , payload   = new Buffer(1300)
 
   beforeEach(function(done) {
+    bufferVal = 0
     port = nextPort()
     server = coap.createServer()
     server.listen(port, done)
   })
 
+  beforeEach(function(done) {
```

Comment:
Copied from 'test/server.js', probably because 'done' passed to both server.listen and client.bind.